docs/Community/Governance.md

```diff
@@ -0,0 +1,161 @@
+* **Open Development**: All ideas and contributions are welcome and accepted based on merit and alignment with project goals.
+* **Open Source**: All project code and dependencies are open source; see contribution guidelines below.
+
+Code of Conduct
+---------------
+
+The Open Enclave SDK community abides by the
+[OE SDK Code of Conduct](../CODE_OF_CONDUCT.md). Here is an
+excerpt:
+
+*In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to make participation in our project and our
+community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and
+expression, level of experience, education, socio-economic status, nationality,
+personal appearance, race, religion, or sexual identity and orientation.*
+
+As a member of the OE SDK project, you represent the project and your fellow
+contributors. We value our community tremendously and want to build and maintain
+a friendly, inclusive, collaborative environment for all our contributors and
+users.
+
+Community Governance Structure
+------------------------------
+
+The Open Enclave SDK community uses a
+[consensus-seeking](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making)
+approach to governance at all levels. To support a growing project, we have
+adopted a loose hierarchy wherein domain-specific decisions are handled within
+each domain, or Special Interest Group (**SIG**), cross-domain work is enabled
+through Working Groups (**WG**), and a Community Governance Committee (**CGC**)
+meets to review the overall structure, ensuring the health of the project.
+
+_NOTE: In lieu of a "technical steering committee", the Architecture SIG should
+be considered as the best forum for technical debates that affect the whole
+project_
+
+Each group (SIGs, WGs, and CGC) shall maintain documentation in this repository.
+Creating a new SIG or WG requires the approval of the CGC, and is formally
+created by the merging of a "charter" document. A SIG is generally responsible
+for the delivery of a technical artefact (code, release binaries, operation of
+infrastructure, etc), whereas a WG is generally not responsible for these types
+of artefacts.
+
+A SIG Chair is responsible for the health of that SIG, and should be considered
+authoritative for that subsection of the project's code base insofar as code or
+other technical deliverables are a responsibility of that SIG. Additionally,
+each SIG is responsible for establishing its own governance process (defining
+the process for SIG Chair selection, for example).
+
+When a change affects code owned by multiple SIGs, it should be coordinated with
+the respective SIGs. If long-term collaboration is anticipated, cross-SIG
+efforts can be coordinated by establishing a Working Group (WG), which is given
+a charter and which reports progress back to the relevant SIGs and/or the CGC.
```

Comment:
Actually, maybe here is the right place to bring up the SIG architecture (lines 102-105); it seems to deserve some special mention because it's effectively a permanent WG across all technical SIGs, that also produces artifacts (design docs etc.)